embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/IntegrationTestRule.kt

```diff
@@ -212,7 +212,11 @@ internal class IntegrationTestRule(
                 deliveryService = FakeDeliveryService(),
             ),
         val startImmediately: Boolean = true
-    )
+    ) {
+        fun logWebView(url: String) {
+            Embrace.getImpl().logWebView(url)
```

Comment:
Why do we need to add this here? Shouldn't the integration test be testing the API by going through Embrace.java in the test?

Reply:
There is no public api to track web view URL, it's captured through the swazzler hook.